admin/services/tech-support/redis.service.js

```diff
@@ -56,7 +56,8 @@ const redisService = {
       /^result:/,
       /^sasToken:/,
       /^schoolData.sqlFindOneById/,
-      /^settings$/
+      /^settings$/,
+      /^ratelimit::/
```

Comment:
This only allows us to delete a single rate-limit entry at a time.  More changes would be needed to add this to the batch-delete feature.

Reply:
this has now been removed as we are not supporting single drops for the rate limit sorted set